backend/ftp/ftp.go

```diff
@@ -53,6 +53,10 @@ func init() {
 			Name:    "tls",
 			Help:    "Use FTP over TLS (Implicit)",
 			Default: false,
+		}, {
+			Name:    "explicit_tls",
+			Help:    "Use FTP over TLS (Explicit)",
```

Comment:
Can you expand the help for `explicit_tls` and `tls` now we have both I think we need to write a bit more here.

Note the Help can stretch over multiple lines - the first line is used for the command line help the next is the long help which will go into the docs and is shown to the user at config time.

Eg something like this for explicit TLS.


```go
    Name:    "explicit_tls",
    Help:    `Use FTP over TLS (Explicit)

When using explicit FTP over TLS the client will connect using TLS
right from the start. This is usually served over port 990 rather than
port 21.`
```

Reply:
Thanks for the tip. I have extended the documentation of both parameters.